### DISCUSSION TITLE
Unable to set cookies in Browser

### DISCUSSION THREAD

**Discussion by nitesh-lab** [vercel/community#6671] [Help]
### Summary

unable to set cookies when deployed but  in localhost it  is working fine and in postman also working i am new to deployment please guide did tried different solutions but did
 not got any working solution

### Steps to Reproduce

![gem3](https://github.com/vercel/community/assets/113165181/305ac122-6a5f-424f-a981-da3317d351a6)
![gem1](https://github.com/vercel/community/assets/113165181/b759c6b8-6b35-4c3e-95a1-8ce91e9f7904)

![gem2](https://github.com/vercel/community/assets/113165181/c9d1bbb4-617d-469f-ae43-973acd5fb2d8)

**Comment by versecafe** [comment]
@nitesh-lab could you share a minimal replication and what framework you are using?

**Comment by nitesh-lab** [comment]
Thanks for Reply. I am using vite-react and node js backend and before trying to deploy my main project i was just testing whether everything works fine.
![gem1](https://github.com/vercel/community/assets/113165181/c1f24113-c281-4477-9de0-352ed83ed6ac)
![gem2](https://github.com/vercel/community/assets/113165181/8f5894bb-e532-4851-88f9-db04e635f805)
![gem3](https://github.com/vercel/community/assets/113165181/23c6f21e-d45e-40c9-8664-9b25c813e436)
i had uploaded both parts.
